src/components/TimeseriesDataExport/stories/basic.story.mdx

```diff
@@ -59,6 +59,8 @@ const TimeseriesChartExportWrapper: React.FC<
 
 <Props of={TimeseriesDataExportFC} />
 
+Please ignore `form*`	and `wrappedComponentRef` props.
```

Comment:
Please ignore `form*`	and `wrappedComponentRef` props. ->
**Note:** Props `form` and `wrappedComponentRef` come from `Form` HOC of antd library and displays here just as a part of the properties interface. You haven't to provide them by yourself.